### ISSUE TITLE
err message: invalid DWARF

### ISSUE THREAD

**Issue by woodard** [dyninst/dyninst#1018]
The GCC 11 DWARF seems to confuse dyninst
running repo1 on  /usr/lib64/libzstd.so.1.4.9 from libzstd-1.4.9-1.fc34.x86_64 using dyninst 11 
yields:
err message: invalid DWARF
err message: invalid DWARF
err message: invalid DWARF

when getting function parameters:
65:    if( i->getParams(lvars) == false){   

This should probably be sorted out. If gcc is in fact generating invalid DWARF please let me know. More likely Dyninst is getting tripped up by the DWARF5 generated by GCC

**Comment by woodard** [comment]
This showed up in 604 of my 1000 tests so it is fairly common.

**Comment by stanfordcox** [comment]
Yes, quite common for gcc 11, which defaults to dwarf 5.  To quiet down the messages (some testsuite tests get this repeatedly) for the Fedora rawhide release I did:
-                cerr << "err message: " << dwarf_errmsg(dwarf_errno()) << endl;
+	        dwarf_printf("(0x%lx) Error while decoding location: %s\n", id(), dwarf_errmsg(dwarf_errno()));
